src/modules/shared.mjs

```diff
@@ -105,92 +76,244 @@ export function addFeedbackLink() {
   document.body.appendChild(aside);
 }
 
-export function updateLogo() {
-  const logo = document.getElementsByTagName("img")[0];
-  logo.src = "https://beta.nj.gov/files/dol_logo.png";
-  logo.style.removeProperty("width");
-  logo.style.removeProperty("height");
-  logo.style.marginLeft = isDesktop() ? "107px" : "18px";
-  logo.alt = "Official logo for the New Jersey Department of Labor";
-  logo.parentElement.parentElement.removeAttribute("height");
+// Used for tagged template strings
+export function html(strings, ...values) {
+  return String.raw({ raw: strings }, ...values);
 }
 
-// Used for tagged template strings
-const html = (strings, ...values) => String.raw({ raw: strings }, ...values);
+export function js(strings, ...values) {
+  return String.raw({ raw: strings }, ...values);
+}
+
+export function css(strings, ...values) {
+  return String.raw({ raw: strings }, ...values);
+}
 
 export const HEADER_HTML = html`
   <header
-    style="padding:12px 18px;background-color:black;color:white;font-size:13px;line-height:150%;margin-bottom:52px"
+    style="padding: 10px 20px; background-color: black; color: white; font-size: 13px; line-height: 150%"
+    id="header"
   >
     <div style="margin-bottom:4px">
       <b>New Jersey Department of Labor and Workforce Development</b>
     </div>
     <a
       href="https://www.nj.gov/labor/myleavebenefits/"
-      style="text-decoration:underline;color:white"
+      style="text-decoration: underline; color: white; text-underline-offset: 2px"
       >myLeaveBenefits.nj.gov</a
     >
   </header>
 `;
 
-export function getRedesignHtml(
-  status,
-  statusExtra,
-  whatsNext,
-  whatsNextExtra
-) {
-  if (!status || !statusExtra) return "";
+export const FOOTER_HTML = html`
+  <footer
+    style="
+      background-color: #eff6fb;
+      padding: 20px 13px;
+      border-top: 1px solid #0b4778;
+      color: black;
+    "
+  >
+    <h3
+      style="
+        margin: 0;
+        font-size: 22px;
+        line-height: 32px;
+        margin-bottom: 8px;
+      "
+    >
+      Need help?
+    </h3>
+    <div
+      style="display: flex; align-items: start; gap: 16px; margin-bottom: 8px"
+    >
+      <img src="./assets/phone.svg" alt="" />
+      <div style="line-height: 21px">
+        <strong>Call</strong> 609-292-7060 | (8:00am - 4:30pm, Monday - Friday)
+        Wait times are shortest Wednesday- Friday
+      </div>
+    </div>
+    <div
+      style="
+        display: flex;
+        align-items: center;
+        gap: 16px;
+        margin-bottom: 8px;
+      "
+    >
+      <img src="./assets/fax.svg" alt="" />
+      <div style="line-height: 21px"><strong>Fax</strong> 609-984-4138</div>
+    </div>
+    <div
+      style="display: flex; align-items: start; gap: 16px; margin-bottom: 8px"
+    >
+      <img src="./assets/mail.svg" alt="" />
```

Comment:
Dust/pebble: adjust so that Mail text is aligned with the rest

![Screenshot_2024-05-01_at_5_49_31_PM-2](https://github.com/newjersey/paid-leave-claim-status/assets/1824854/91cec3b8-2b38-40e4-952d-87759e2787cf)

Reply:
Good catch! Fixed with a CSS grid